client/src/components/TreeImageCard/TreeImageCard.js

```diff
@@ -0,0 +1,113 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import compose from 'recompose/compose'
+import { connect } from 'react-redux'
+import Typography from '@material-ui/core/Typography'
+import { withStyles } from '@material-ui/core/styles'
+import Card from '@material-ui/core/Card'
+import CardActions from '@material-ui/core/CardActions'
+import CardContent from '@material-ui/core/CardContent'
+import CardMedia from '@material-ui/core/CardMedia'
+import Button from '@material-ui/core/Button'
+
+import { selectedHighlightColor } from '../../common/variables.js'
+
+const styles = theme => ({
+  cardImg: {
+    width: '100%',
+    height: 'auto'
+  },
+  cardTitle: {
+    color: '#f00'
+  },
+  card: {
+    cursor: 'pointer',
+    margin: '0.5rem',
+    border: `2px #eee solid`
+  },
+  selected: {
+    border: `2px ${selectedHighlightColor} solid`
+  },
+  cardMedia: {
+    height: '12rem'
+  },
+  cardWrapper: {
+    width: '33.33%',
+  }
+})
+
+
+class TreeImageCard extends Component {
+
+  constructor(props) {
+    super(props)
+  }
+
+  onStatusToggle = (e, id, isActive) => {
```

Comment:
@deepwinter it is un-deleted over here

Reply:
I guess I am wondering if this is the right pattern, including the toggle callback in the TreeCard?   I know we also toggle active from the list view.

Reply:
That file is not referenced anywhere.

Reply:
> 
> 
> That file is not referenced anywhere.

okay, will leave this alone until you have the PR where you want it